### ISSUE TITLE
ARIA7/9: Specify IE9 and IE10

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#149]
For instance, In <a href="https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/ARIA7.html#ARIA7-ex2">example 2 of ARIA7</a>:
> ... it merely serves to ensure that some browsers (IE9, IE10) will include the span element in the accessibility tree ...

At the present time, What is the status of other browsers?

**Comment by joshueoconnor** [comment]
@momdo We've discussed this on the working group call, and think it would be good if you could do some testing and make a suggestion for us to update the technique. Would this be something you are interested in?

**Comment by momdo** [comment]
@joshueoconnor 
I just meant that obsolete information should be updated/deleted or otherwise explicitly marked as obsolete...

**Comment by joshueoconnor** [comment]
The group does endeavour to keep our techniques up to date - closing.
